### DISCUSSION TITLE
webpack deleted all project files, including itself

### DISCUSSION THREAD

**Discussion by vojjin-gmail** [webpack/webpack#17876] [Q&A]
I was planing to nbetter organize js files, and to pack them with a webpack. I have installed it and tried several cofigurations to run but received just empty js file as a result. However, on one `npm run build` command ... First I receive an error, but since my folder was opened in a another window, I was just looking files and folders disappaearing! complete project was deleted, and there was created empty folder `dist` .  Here is the log of action:

..ts/php/goran/boxold [main|●2✚1…1] ❯❯❯ npm run build

> boxold@1.0.0 build
> webpack

node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module 'schema-utils'
Require stack:
- /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/lib/index.js
- /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/lib/webpack-cli.js
- /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/lib/bootstrap.js
- /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/bin/cli.js
- /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/bin/webpack.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at get ValidationError (/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/lib/index.js:331:10)
    at Function.ValidationError (/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/lib/util/memoize.js:22:13)
    at WebpackCLI.isValidationError (/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/lib/webpack-cli.js:1771:46)
    at /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/lib/webpack-cli.js:1787:39
    at /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/lib/webpack.js:157:8
    at /Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/lib/HookWebpackError.js:68:3 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/lib/index.js',
    '/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/lib/webpack-cli.js',
    '/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/lib/bootstrap.js',
    '/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack-cli/bin/cli.js',
    '/Users/vojinpetrovic/projects/php/goran/boxold/node_modules/webpack/bin/webpack.js'
  ]
}

Node.js v18.17.0
~/projects/php/goran/boxold ❯❯❯ ls
dist
~/projects/php/goran/boxold ❯❯❯ 

Luckily for me I was pushed to git before this action so I was able to return files. Please check what can cause this behaviour.

**Comment by alexander-akait** [comment]
I think you have globally installed webpack-cli, check it out
